jooby/src/test/java/org/jooby/RequestLoggerTest.java

```diff
@@ -147,6 +169,13 @@ private Block path(final String path) {
     };
   }
 
+  private Block query(final String query) {
+    return unit -> {
+      Request req = unit.get(Request.class);
+      expect(req.queryString()).andReturn(Optional.of(query)).atLeastOnce();
```

Comment:
You could move the queryString() as part of the if block ... but I'm OK with this.